app/controllers/profiles_controller.rb

```diff
@@ -2,8 +2,7 @@ class ProfilesController < ApplicationController
   before_action :require_login, :allow_anyone
 
   def show
-    @checkouts = current_user.checkouts.includes(:book)
-
+    @checkouts = current_user.checkouts.includes(:book).where(closed_at: nil)
```

Comment:
I would extract a scope for the `where(closed_at: nil)`. Something like `closed` maybe?